cmd/certs/main.go

```diff
@@ -67,7 +67,7 @@ const (
 	defVaultHost    = ""
 	defVaultRole    = "mainflux"
 	defVaultToken   = ""
-	defVaultPKIPath = "pki_int"
+	defVaultPKIIntPath = "pki_int"
```

Comment:
fix the tabulation

Reply:
@manuio this looks well aligned to me

Reply:
@drasko The golang linter should align `=` characters.